### ISSUE TITLE
Possibility to reactivate a deactivated account yourself

### ISSUE THREAD

**Issue by mritzmann** [nextcloud/preferred_providers#5]
We receive support-requests from users who have already uploaded files and set up calendar and contacts, but have not activated the account. I see several ways and problems:

* I can activate the account by hand. But then I have no certainty that the e-mail address is correct.
* I can delete the account and tell the user that they can re-register.
* Both possibilities cause a manual overhead. The customer must actively contact us.

In my opinion it needs an option so that the user can send himself a new verify link. Maybe with a resend-email-message after a failed login. Or via password reset.

**Comment by NesooseN** [comment]
Yes, this would be nice, we also get huge amount of time resending or approving users because of this problem.
